src/Tooltip/TooltipTouch.tsx

```diff
@@ -80,40 +94,42 @@ export default class TooltipTouch extends React.PureComponent {
     });
   }
 
-  getLocals({ popover, ...props }) {
+  render() {
+    const { popover: _popover, children, className, id, style, type, size } = this.props as TooltipTouchDefaultedProps;
     const { isOpen } = this.state;
-
-    return {
-      ...props,
+    const popover = Object.assign({
```

Comment:
I don't understand the need of `Object.assign` here

Reply:
it doesn't make sense at all ooops 💃